scss/_nav.scss

```diff
@@ -35,6 +35,25 @@
   }
 }
 
+.nav.nav-pills.flex-column > li {
```

Comment:
Maybe doing the selector inheritance with sass would help with the linting errors here (?).

i.e. something like this:
```
.nav {
  .nav-pills {
    .flex-column > li {
```